### PR TITLE
student result fragment completed

### LINKED ISSUE #43: Student Result Fragment Implementation
You have to Implement the result fragment same as:- 

Screenshot 2022-04-15 at 10 46 52 PM

### PR DESCRIPTION
**Pull request by ismailbohra**
task completed
Screenshot_20220415_171305
Screenshot_20220415_171337
Screenshot_20220415_171353

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:7.0.4"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20"
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30'
```
Comment (MohitGupta121) [inline build.gradle]: Why you changing build file change it according to development branch.
Not add your local files changes

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by ismailbohra** [comment]
gradle updated

**Review by MohitGupta121 — APPROVED** [review]
Thanks for your PR.

**Comment by MohitGupta121** [comment]
@ismailbohra  upstream your fork repo

**Pull request merged**